packages/next/client/future/image.tsx

```diff
@@ -645,6 +726,7 @@ const ImageElement = ({
         height={heightInt}
         decoding="async"
         data-nimg="future"
+        {...(fill ? { 'data-nfill': true } : {})}
```

Comment:
Lets use `data-nimg="future-fill"` instead?

Reply:
Sounds fine to me--changed